bulma-rtl.sass

```diff
@@ -0,0 +1,10 @@
+/*! bulma.io v0.4.0 | MIT License | github.com/jgthms/bulma */
+@charset "utf-8"
+
+$right: left
+$left: right
+
+html
+  direction: rtl
+
+@import "bulma"
```

Comment:
Both versions don't sound so pretty to me. Perhaps there's another way?

In the future, the solution would be that CSS for both directions is included and is chosen using the [`:dir`](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir) pseudo selector.

Right now, what we could possibly do is similar, with the difference of selecting on `html[dir=rtl]`. How do you feel about that?

- Most RTL pages on the web will have `<html dir="rtl">`, right?
- Bulma is CSS for components. Components in an RTL page will always be RTL, right? There's no common reason to mix both RTL and LTR styled components, I think.
- Bulma has a component for styling content. I don't think it has any directionality-relevant properties, right? It is probably simmetrical.

So, if Bulma would "just work" for RTL when the HTML element has `dir="rtl"` perhaps that would be best?

If a slight increase in size is an issue, that could be resolved by splitting it into some imports. But that would create so many new files... I think not worth it.

Thoughts?